release/release.go

```diff
@@ -117,3 +124,10 @@ type GcpImage struct {
 	Family  string `json:"family,omitempty"`
 	Name    string `json:"name,omitempty"`
 }
+
+// IBMCloudObject represents an IBMCloud/PowerVS cloud object - which is a cos bucket image
+type IBMCloudObject struct {
+	Object string `json:"object,omitempty"`
+	Bucket string `json:"bucket,omitempty"`
+	Url    string `json:"url,omitempty"`
```

Comment:
To be clear, is there information in the URL which can't be derived from the bucket and object?